qoffer-infrastructure/src/main/resources/offer-template/offer.html

```diff
@@ -15,27 +15,27 @@
     <div class="grid-container">
         <div class="header"><img src="offer_header.png"/></div>
         <div class="customer">
-            <div class="addressee">Dr. Max Mustermann</div>
-            <div class="description">Quantitative Biology Center</div>
-            <div class="street">Auf der Morgenstelle 10</div>
-            <div class="zipcode">72076</div>
-            <div class="town">Tübingen</div>
-            <div class="country">-</div>
+            <div id="cName">Dr. Max Mustermann</div>
+            <div id="cOrganisation">Quantitative Biology Center</div>
+            <div id="cStreet">Auf der Morgenstelle 10</div>
+            <div id="cPostalCode">72076</div>
+            <div id="cCity">Tübingen</div>
+            <div id="cCountry">-</div>
         </div>
         <div class="qbic-affiliation">
             <div class="qbic-ad">Quantitative Biology Center (QBiC) <br> and 
                 Technology Platforms</div>
             
             <div class="small-spacer"></div>
             <div class="info-title">Project manager</div>
-            <div id="project-manager">Dr. Doctore Doctores</div>
+            <div id="pmName">Dr. Doctore Doctores</div>
```

Comment:
`pm` is generally used as an abbreviation for different things like `private message(ing/er)`, `10pm`, `particulate matter`, `postmaster`. I would like to keep the template explicit on what is expected here therefore my suggestion. Also, I would prefer to keep the HTML/CSS syntax, like our code style, to google standard which suggests hypens https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters
```suggestion
            <div id="project-manager-name">Dr. Doctore Doctores</div>
```

Reply:
html layout task

Reply:
The google html css guide is linked to the trello card.